near-plugins/tests/contracts/access_controllable/src/lib.rs

```diff
@@ -21,6 +23,26 @@ pub struct StatusMessage {
 
 #[near_bindgen]
 impl StatusMessage {
+    // Initially adding (super-)admins is done via internal methods, for
+    // example:
+    //
+    // ```
+    // self.__acl.add_super_admin_unchecked(account_id);
+    // self.__acl.add_admin_unchecked(role, account_id);
+    // ```
+    //
+    // **Attention**: Contracts should call `__acl.*_unchecked` methods only
+    // from within methods with attribute `#[init]` or `#[private]`.
+    //
+    // Once an account is (super-)admin, it may add other admins and grant
+    // roles.
+    //
+    // If needed, It's also possible to grant a role without checks:
+    //
+    // ```
+    // self.__acl.grant_role_unchecked(role, account_id);
+    // ```
+
```

Comment:
Due to removing `acl_*_unchecked` methods from the trait.

I'm not really comfortable with asking contract developers to call `__acl.*_unchecked` to add initial (super-)admins. Instead, we could add some method like the following to the trait `AccessControllable`:

```rust
/// Makes the contract a super-admin.
/// 
/// It is #[private] in the implementation provided by this trait, i.e. only the
/// contract itself may call this method.
fn acl_make_self_super_admin();
```